lib/pure/terminal.nim

```diff
@@ -16,6 +16,33 @@
 ## Similarly, if you hide the cursor, make sure to unhide it with
 ## `showCursor` before quitting.
 
+##[
+## Playing with colorful and styled text
+]##
+
+## Procs like `styledWriteLine`, `styledEcho` etc. have a temporary effect on
+## the terminal. Style parameters only affect the text parameter right after them.
+## While `setBackGroundColor`, `setForeGroundColor` etc. have a lasting
+## influence on the terminal, you can use `resetAttributes` to
+## reset the current style of the terminal.
```

Comment:
```suggestion
## reset the default style of the terminal.
```

can you double check which one is correct, and clarify accordingly ? IIUC terminal state is not stack based and you can't reset previous state, instead you set default state (see `ansiResetCode* = "\e[0m"`)

Reply:
My mistake. You are right.  I changed the sentence from "clear the current style of the terminal" rashly.